### PR TITLE
Added two public helper functions.

### PR DESCRIPTION
**Pull request by Hassaan**
First function is a simple 'isShowing()' method and the second returns the number of messages still waiting in the queue to be displayed to the user.

### CODE REVIEW

File: library/src/net/simonvt/messagebar/MessageBar.java [hunk 1]
```diff
@@ -164,6 +164,18 @@ public void clear() {
         mMessages.clear();
         mHideRunnable.run();
     }
+    
+    public boolean isShowing(){
```
Comment (JakeWharton) [inline library/src/net/simonvt/messagebar/MessageBar.java]: Space before `{`.

File: library/src/net/simonvt/messagebar/MessageBar.java [hunk 2]
```diff
@@ -164,6 +164,18 @@ public void clear() {
...
+        return mShowing;
+    }
+    
+    /**
+     * Returns the total number of messages still waiting in the queue to be 
+     * shown to the user.
+     */
+    public int getMessagesLeft(){
```
Comment (JakeWharton) [inline library/src/net/simonvt/messagebar/MessageBar.java]: Space before `{`.

File: library/src/net/simonvt/messagebar/MessageBar.java [hunk 3]
```diff
@@ -164,6 +164,18 @@ public void clear() {
...
+    	return mMessages.size();
```
Comment (JakeWharton) [inline library/src/net/simonvt/messagebar/MessageBar.java]: Needs indented two more spaces.

File: library/src/net/simonvt/messagebar/MessageBar.java (hunk 2 above)
Comment (JakeWharton) [inline library/src/net/simonvt/messagebar/MessageBar.java]: This should also probably be named something a bit more formal. `getMessageCount`, perhaps?

Comment (Hassaan) [inline library/src/net/simonvt/messagebar/MessageBar.java]: Yes I think you are right. Initially I was planning of naming it `getMessagesLeftInQueue`, but that seemed a bit too long for a method name. What would you suggest be a more formal name?

Comment (SimonVT) [inline library/src/net/simonvt/messagebar/MessageBar.java]: `getMessageCount` sounds good

Comment (Hassaan) [inline library/src/net/simonvt/messagebar/MessageBar.java]: Yes that sounds much better actually.

**Comment by gabrielittner** [comment]
@Hassaan Could you update the pull request? I'd really like to have those two methods in the library.

**Comment by gabrielittner** [comment]
@SimonVT Could you merge this?

**Comment by Hassaan** [comment]
Looks like the library has improved much over the year and this request is not necessary anymore.

**Pull request closed without merging**